src/blocks/media-block/components/save.js

```diff
@@ -0,0 +1,79 @@
+import { Component } from "@wordpress/element";
+import cls from 'classnames';
+
+export class MediaBlockSaveComponent extends Component{
+
+    render () {
+
+        const { attributes } = this.props;
+
+        const isLinkAvailable = attributes.url !== undefined && attributes.url !== "";
+
+        let titleText = isLinkAvailable ?
+            <a href={ attributes.url } className={"c-block__title-link u-theme--link-hover--dark"}> {attributes.title} </a> :
+            attributes.title;
+
+        const styles = cls(
+            'c-media-block c-block l-grid-wrap',
+            {'c-block__row l-grid-wrap--5-of-7': attributes.alignment === "left"},
+            {'c-block__stacked l-grid-wrap--6-of-7': attributes.alignment === "center"}
+        );
+
+        const imageStyles = cls(
+            'c-block__image',
+            {'l-grid-item--s--1-col l-grid-item--m--1-col l-grid-item--l--2-col l-grid-item--xl--1-col u-padding--zero--sides': attributes.alignment === "left"},
+            {'l-grid-item--s--6-col l-grid-item--m--4-col l-grid-item--xxl--3-col': attributes.alignment === "center"}
```

Comment:
Replace line 25 with the following: `{'l-grid-item--s--6-col l-grid-item--m--4-col l-grid-item--l--3-col': attributes.alignment === "center"}`

Reply:
It was updated.

Thanks